src/chrome/content/rules/FreeCallsToUSA.co.uk.xml

```diff
@@ -0,0 +1,13 @@
+<ruleset name="FreeCallsToUSA.co.uk">
+	<target host="freecallstousa.co.uk" />
+	<target host="www.freecallstousa.co.uk" />
+
+	<test url="http://freecallstousa.co.uk/" />
+	<test url="https://freecallstousa.co.uk/" />
+	<test url="http://www.freecallstousa.co.uk/" />
+
+	<rule from="^https?://freecallstousa\.co\.uk/" to="https://www.freecallstousa.co.uk/" />
```

Comment:
You should not touch https domains and forcing to www as long as this works server side.